README.md

```diff
@@ -105,7 +105,7 @@ Support this project by donation, help Badgen continue and evolving!
 
 ## Sponsors
 
-<a href="https://zeit.co"><img src="https://sponsors.badgen.now.sh/zeit.svg" height="60px" /></a>
+<a href="https://vercel.com"><img src="https://sponsors.badgen.now.sh/zeit.svg" height="60px" /></a>
```

Comment:
The logo needs to be updated too. See https://vercel.com/design

Reply:
@styfle I'm planning to create a hand organized full width image for sponsors, make it more eye-catching. This PR is all good now :D